cgi-bin/DW/External/User.pm

```diff
@@ -69,13 +70,17 @@ sub ljuser_display {
     my $display_class = $opts{no_ljuser_class} ? "" : " class='ljuser'";
     my $domain = $self->site->{domain} ? $self->site->{domain} : $self->site->{hostname};
 
+    $nolink = 1 if exists $DW::External::Site::deadsites{$domain};
+
+    $nolink = $nolink || $opts{no_link};
+
     return
-          "<span style='white-space: nowrap;'$display_class>"
-        . ( $opts{no_link} ? '' : "<a href='$profile_url'>" )
+        "<span style='white-space: nowrap;'$display_class>"
+        . ( $nolink ? '' : "<a href='$profile_url'>" )
         . "<img src='$badge_image->{url}' alt='[$domain profile] ' style='vertical-align: text-bottom; border: 0; padding-right: 1px;' width='$badge_image->{width}' height='$badge_image->{height}'/>"
-        . ( $opts{no_link} ? '' : "</a><a href='$journal_url'>" )
+        . ( $nolink ? '' : "</a><a href='$journal_url'>" )
         . "<b>$user</b>"
```

Comment:
Revisiting the original issue, it looks like you should be printing something like `"<b>$user [$site]</b>" ` here if `exists $DW::External::Site::deadsites{$domain}` is true?